src/serlo-editor-repo/plugin-text/components/math-element.tsx

```diff
@@ -37,6 +37,20 @@ export function MathElement({
   const selected = useSelected()
   const preferences = useContext(PreferenceContext)
 
+  const isInsideListElement = useMemo(() => {
+    const path = ReactEditor.findPath(editor, element)
+    const ancestorNodes = [...Node.ancestors(editor, path)]
+    return ancestorNodes.some((ancestor) => {
+      const ancestorPath = ancestor[1]
+      const ancestorNode = Node.get(editor, ancestorPath)
+      return (
+        'type' in ancestorNode &&
+        (ancestorNode.type === 'ordered-list' ||
+          ancestorNode.type === 'unordered-list')
+      )
+    })
+  }, [editor, element])
+
```

Comment:
```typescript
import { isOrderedListActive, isUnorderedListActive } from '../utils/list'
```

```suggestion
  const isInsideListElement = useMemo(() => {
    return isOrderedListActive(editor) || isUnorderedListActive(editor)
  }, [editor])
```

Just tried it, seems to work. What do you think?

Reply:
Nice suggestion! I see the potential to reuse or combine the code. However, looking into isOrderedListActive I find myself putting more work into understanding what it does than the current solution. For example, I have to additionally think about selection stuff. I'll try to figure something out that is more clear, but inspired by your suggestion.